plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftConnectorTest.java

```diff
@@ -121,6 +122,24 @@ public void testSuperColumnType()
         }
     }
 
+    @Test
+    public void testVarcharZero()
```

Comment:
note that not all redshift tests run on CI since some classes are excluded in surefire config. So don't be surprised if CI still appears green.

(even before that tests for things which require secrets like Redshift don't run for PRs submitted from forks - a maintainer needs to explicitly invoke such tests).

Reply:
Thanks. That's very useful. I didn't know that, so your guess was right to guide me here :)